tests/backup/backup_rancher_test.go

```diff
@@ -265,3 +271,534 @@ var _ = Describe("{SingleNamespaceBackupRestoreToNamespaceInSameAndDifferentProj
+				destProjectList = append(destProjectList, destProject)
+				destProjectID, err := Inst().S.(*rke.Rancher).GetProjectID(destProject)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Getting project Id for project: %s in destination cluster", destProject))
+				destProjectIDList = append(destProjectIDList, destProjectID)
+			}
+			err = SetSourceKubeConfig()
+			log.FailOnError(err, "Switching context to source cluster failed")
+		})
+
+		Step("Restoring the backup taken with namespace and project mapping", func() {
+			log.InfoD("Restoring the backup taken with namespace and project mapping")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for i, app := range appNamespaces {
+				restoreNamespace := fmt.Sprintf("restore-%v-%v-%v", app, RandomString(5), i)
+				namespaceMapping[appNamespaces[i]] = restoreNamespace
+				restoreNamespaceList = append(restoreNamespaceList, restoreNamespace)
+			}
+			projectNameMapping[sourceProject] = destProjectList[0]
+			projectUIDMapping[sourceProjectID] = destProjectIDList[0]
+			restoreName = fmt.Sprintf("%s-%v-default", restoreNamePrefix, RandomString(5))
+			restoreList = append(restoreList, restoreName)
+			err = CreateRestoreWithProjectMapping(restoreName, backupName, namespaceMapping, destinationClusterName, orgID, ctx, nil, projectUIDMapping, projectNameMapping)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Creating restore- %s from backup %s", restoreName, backupName))
+		})
+
+		Step("Restore the backup taken with replace policy and move destination ns from one project to another project while restoring", func() {
+			log.InfoD("Restore the backup taken with replace policy and move ns from one project to another project while restoring")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			restoreName = fmt.Sprintf("%s-replace-ns-project-move-%v", restoreNamePrefix, RandomString(5))
+			restoreList = append(restoreList, restoreName)
+			err = CreateRestoreOnRancherWithoutCheck(restoreName, backupName, namespaceMapping, destinationClusterName, orgID, ctx, nil, projectUIDMapping, projectNameMapping, 2)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Creating restore- %s with replace policy from backup %s", restoreName, backupName))
+			log.Infof("Move namespaces in destination cluster to diff project while restore is going on")
+			err = Inst().S.(*rke.Rancher).ChangeProjectForNamespace(destProjectList[1], restoreNamespaceList)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Moving namespaces %v from project %s to project %s", restoreNamespaceList, destProjectList[0], destProjectList[1]))
+			log.Infof("Verifying if restore is successful after moving destination namespaces from one project to another")
+			err = restoreSuccessCheck(restoreName, orgID, maxWaitPeriodForRestoreCompletionInMinute*time.Minute, restoreJobProgressRetryTime*time.Minute, ctx)
```

Comment:
Is it not restoreJobProgressRetryTime*time.Second and not restoreJobProgressRetryTime*time.Minute?

Reply:
Value of restoreJobProgressRetryTime is 1.
Checking after every 1 sec will not make sense.
@kphalgun-px ?

Reply:
My bad. I assumed the retry intervals are usually set in seconds.